### PR TITLE
feat(charts) add alternative rolling average rendering mode

### PR DESCRIPTION
**Pull request by zegl**
The rendered chart looks like this: image

### CODE REVIEW

File: web.go [hunk 1]
```diff
@@ -196,15 +212,13 @@ func (wr *Routes) charts(w http.ResponseWriter, r *http.Request) {
...
-	for _, sample := range items {
-		ts.XValues = append(ts.XValues, sample.When)
-		ts.YValues = append(ts.YValues, sample.Duration.Seconds())
+	var ts chart.TimeSeries
+
+	switch mode {
+	case "rolling-average":
+		ts = rollingAverageTs(items)
+	default:
+		ts = allBuildsTs(items)
```
Comment (JensRantil) [inline web.go]: Did you consider adding the rollingAverageTs as yet another time series in the same graph? That is, simply add it to https://github.com/tink-ab/buildkite-stats/pull/1/files#diff-a3520fb65206c9e232f10baaff310c7aR228 array. Would make it easier to both look at long-term trend and short-term trends at the same time.

Comment (zegl) [reply web.go]: I did. It looked like shit. 😞

Comment (JensRantil) [reply web.go]: ✅

File: web.go [hunk 2]
```diff
@@ -30,6 +33,12 @@ func (wr *Routes) Routes() chi.Router {
 }
 
 func (wr *Routes) root(w http.ResponseWriter, r *http.Request) {
+
+	chartMode := "all"
+	if r.RequestURI == "/rolling-average" {
+		chartMode = "rolling-average"
+	}
```
Comment (JensRantil) [inline web.go]: Maybe I've missed something, but how about adding a link on the frontpage to enable rolling average?

Comment (zegl) [reply web.go]: The link is here: https://github.com/tink-ab/buildkite-stats/pull/1/files#diff-a3520fb65206c9e232f10baaff310c7aR149

**Pull request merged**